app/views/layouts/messages.html.erb

```diff
@@ -3,18 +3,17 @@
 
   <div class="flex h-screen"
       data-controller="click-toggle"
-      data-click-toggle-flippable-class="md:flex"
-      data-action="keydown.meta+shift+s@document->click-toggle#handleEvent
-                   keydown.meta+period@document->click-toggle#handleEvent"
+      data-click-toggle-flippable-class="!hidden"
   >
 
     <div id="left-column"
-        class="flex-col h-screen w-[260px] bg-black hidden md:block overflow-y-auto overflow-x-clip scrollbar-hide"
+        class="flex-col h-screen w-[260px] bg-black hidden md:flex overflow-y-auto overflow-x-clip scrollbar-hide"
```

Comment:
Oh this is probably what broke the sidebar. The reason I switched it to block is, there aren't > 1 children of this div (or there weren't -- I can't tell from the diff if that's still true) so a flex is useless. Maybe it doesn't hurt? I don't really know if there's a performance hit... 👍 anyway

Reply:
Actually, you're right this can be block. I tried changing this back to flex in my various attempts to debug what was wrong and didn't revert it.

The issue was that the data-click-toggle-class was referencing "md:flex" so changing that to "md:block" required changing the stimulus configuration. That's now been changed as part of the refactor I did so this should be block again.